lib/bios/menus.robot

```diff
@@ -226,9 +226,13 @@ Get Index Of Matching Option In Menu
     [Documentation]    This keyword returns the index of element that matches
     ...    one in given menu
     [Arguments]    ${menu_construction}    ${option}    ${ignore_not_found_error}=${FALSE}
+    Set Suite Variable    ${MATCHES}    ${FALSE}
     FOR    ${element}    IN    @{menu_construction}
-        ${matches}=    Run Keyword And Return Status
-        ...    Should Match    ${element}    *${option}*
+        # Ignore if ${option} is in quotes
+        ${is_in_quotes}=    Run Keyword And Return Status    Should Match    ${element}    *\'${option}\'*
```

Comment:
This solution is valid for a very specific case only. It fixes just one of the symptoms, not the underlying problem.

Have you tried to use `opt_only` argument set to `TRUE` for call to `Enter Submenu From Snapshot And Return Construction` in `Set UEFI Option` (and, if it works, `Get UEFI Option`)? If I understand that code correctly, it should skip the informative lines from menu construction.

If that works, we may even consider changing the default in`Enter Submenu From Snapshot And Return Construction`, assuming it won't break anything else.

Reply:
Because
> I have tried solutions you suggested - but they did not change anything.
I have added a more precise regex for detecting lines containing options in `Get Index Of Matching Option In Menu` instead of changing `Enter Submenu From Snapshot And Return Construction`
https://github.com/Dasharo/open-source-firmware-validation/pull/580/commits/b72630e8dadf673f020969f8803813b114d33c37#diff-6ad7c5fb97cb1099c7ecd42f15dde3caf06d38b9fe5244afb96394454b6d6bfaR232-R235

Reply:
I will check if using the `Check If Menu Line Is An Option` instead wouldn't be better

Reply:
> Have you tried to use opt_only argument set to TRUE

I have tried this suggestion and it seems it works perfectly on a z790-ddr5. 
[me-neuter.robot_log.zip](https://github.com/user-attachments/files/17991477/me-neuter.robot_log.zip)

There is an issue with the me-neuter test suite, where it assumes that the ME lspci entry exists after disabling it using the HAP bit. It *sometimes* does, for example on protectli VP2420 if I recall correctly, but it certainly is not always the case. Created an issue for that
https://github.com/Dasharo/open-source-firmware-validation/issues/621